umh.docs.umh.app/content/en/docs/production-guide/administration/normalize-kafka-topics.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Normalize Kafka Topics"
```

Comment:
The title should use the word "merge_point" as we have been using it so far, and to be honest I did not know what I expected when I clicked on "Normalize Kafka topics". If it is called like "Use the merge point to normalize kafka topics" it could be more identifyable for our team.